Added re-enable animations option

LoneDev6 opened the pull request:
Requested at #49

LoneDev6 committed:
Update idle.js

Added code to hide game graphics when user clicked on disable animations button (You'll only see the bot GUI then)

LoneDev6 committed:
Merge pull request #1 from LoneDev6/LoneDev6-patch-1

Update idle.js

LoneDev6 committed:
Added re-enable animations option

ensingm2 commented:
Nice! I need to make a couple small changes and test, but if this works correctly I'll merge it in after work today.

LoneDev6 committed:
GUI small improvements

Added in GUI:
- score that will be earned at game end
- score to reach to level up

LoneDev6 committed:
GUI small improvements

Added in GUI:
- score that will be earned at game end
- score to reach to level up

LoneDev6 committed:
Merge pull request #2 from LoneDev6/LoneDev6-patch-2

GUI small improvements

LoneDev6 commented:
Thanks a lot :D

ensingm2 committed:
Changed disable_animations to toggle_animations

ensingm2 committed:
Fix for Chrome & Small windows

.prop() wasn't working in chrome, and if your window was sized small enough that the canvas was scaled, neither Chrome of FF would work.

ensingm2 committed:
Moved zone exp up a line

Put it with difficulty

ensingm2 committed:
Add ability to swap planets with anims disabled

Simply restart anims then revert to previous anim state after switch.

ensingm2 committed:
Switched GUI option to "Hide Game"

I know if we left it at "Disable Animations" I'd get at least 2-3 people asking me about why the game suddenly went away and 'broke'. Hopefully this makes it more clear.
